### PR TITLE
pypi inited

### PR DESCRIPTION
**Pull request by pychik**

### CODE REVIEW

**Comment by pychik** [comment]
changes
client -> lightning
swagger_client -> lightning_adapter

**Comment by pychik** [comment]
changes
client -> lightning
swagger_client -> lightning_adapter

also fixes #10 #11

File: generated/.gitignore [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Byte-compiled / optimized / DLL files
```
Comment (vania-pooh) [inline generated/.gitignore]: Do we actually need this file under source control? This file is generated.

File: generated/.swagger-codegen-ignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Swagger Codegen Ignore
```
Comment (vania-pooh) [inline generated/.swagger-codegen-ignore]: Useless file.

File: .gitignore [hunk 3]
```diff
@@ -42,9 +42,12 @@ venv/
 .idea/
```
Comment (vania-pooh) [inline .gitignore:42]: Need to exclude everything under `generated` completely.

File: ci/test.sh [hunk 4]
```diff
@@ -2,23 +2,21 @@
...
-   -i /home/runner/work/lightning_py/lightning_py/selenium.yaml \
+   -i https://raw.githubusercontent.com/aerokube/selenium-openapi/master/selenium.yaml \
    -l python \
-   -o /home/runner/work/lightning_py/lightning_py/python
-
+   -o /home/runner/work/lightning_py/lightning_py/generated \
+   -D packageName=lightning_adapter
 # setup swagger API
-pushd python/ && python setup.py install && cd ..
+pushd generated/ && python setup.py install && cd ..
```
Comment (vania-pooh) [inline ci/test.sh]: Need to do `popd` instead of `cd ..`

**Review by vania-pooh — CHANGES_REQUESTED** [review]
Need to exclude `generated` directory.

File: ci/test.sh (hunk 4 above)
Comment (pychik) [reply ci/test.sh]: fixed in f7c051faca999bf10f026e76334164357339ac52

File: .gitignore (hunk 3 above)
Comment (pychik) [reply .gitignore:42]: removed in f7c051faca999bf10f026e76334164357339ac52

File: generated/.gitignore (hunk 1 above)
Comment (pychik) [reply generated/.gitignore]: removed in f7c051faca999bf10f026e76334164357339ac52

File: generated/.swagger-codegen-ignore (hunk 2 above)
Comment (pychik) [reply generated/.swagger-codegen-ignore]: fixed in f7c051faca999bf10f026e76334164357339ac52

**Review by vania-pooh — APPROVED** [review]

**Comment by vania-pooh** [comment]
Merging. However not sure how this is related to pypi. :)

**Pull request merged**